# Raise an exception from another exception 3.x.
def raise_exception_from(exception, original_exception):
    raise exception from original_exception
